test/test_helper.rb

```diff
@@ -49,20 +63,48 @@ def skip_on_travis
   end
 
   def skip_on_ci
-    skip if ENV["APPVEYOR"] or ENV["TRAVIS"]
+    skip if runs_on_ci?
+  end
+
+  def runs_on_ci?
```

Comment:
Ruby doesn't usually have an `s` at the end of verbs (e.g. `equal?`). `run_on_ci?`, `on_ci?`, simply `ci?`? Not sure